src/Stripe.net/Infrastructure/Extensions/BaseOptionsExtensions.cs

```diff
@@ -1,16 +1,17 @@
 namespace Stripe.Infrastructure.Extensions
 {
+    using System.Net.Http;
     using Stripe;
     using Stripe.Infrastructure.FormEncoding;
 
     internal static class BaseOptionsExtensions
     {
-        /// <summary>Serializes the parameters with form encoding.</summary>
+        /// <summary>Serializes the parameters as a query string.</summary>
         /// <param name="options">The options class containing the parameters.</param>
         /// <returns>A form-encoded string represensation of the parameters.</returns>
         public static string ToQueryString(this BaseOptions options)
         {
-            return FormEncoder.EncodeOptions(options);
+            return FormEncoder.CreateQueryString(options);
```

Comment:
I swear this was the name months ago and we changed it because we did not like it. Likely misremembering lol